recipe_batches/recipe_batches.py

```diff
@@ -4,14 +4,23 @@
 
 
 def recipe_batches(recipe, ingredients):
-    batches = 0
+  # return min(int(ingredients.get(elem, 0) / num) for elem, num in recipe.items())
+    batches = []
     if len(recipe) != len(ingredients):
-        return batches
+        return 0
+
+    for el in recipe:
+        if recipe[el] <= ingredients[el]:
+            batches.append(int(ingredients[el]/recipe[el]))
```

Comment:
So you did do the ratio! I'm glad that helped you out. And way to cast the right type, I'm very happy to see that you have this instinct it will serve you well in the future.